traffic_ops/app/db/migrations/2023121910512261_add_comment_parameter.down.sql

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
The timestamp on this migration, `2023121910512261`, is 10 months old. Would you please give it a more recent timestamp (for example, `2024101800000000`)?